lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -449,6 +453,7 @@ def swap_changed_attributes
           end
 
           def check_scope
+            return unless act_as_list_run_callbacks?
```

Comment:
Since callbacks being disabled is a special condition, can you please flip the conditional here to check if they have been disabled?

`return if act_as_list_callbacks_disabled?`